sites/gent/ch_hpc_policies.tex

```diff
@@ -1,4 +1,85 @@
 \chapter{HPC Policies}
 \label{ch:hpc-policies}
 
-Stub chapter
+Everyone can get access and use the HPC-UGent supercomputing infrastructure and services.
+The conditions that apply depend on your affiliation.
+
+\section{Access for staff & academics}
+
+\subsection{Researchers and staff affiliated with Flemish university associations}
+
+\begin{itemize}
+  \item Includes externally funded researchers registered in the personnel database (FWO, SBO, VIB, IMEC, etc.)
+  \item Includes researchers from all VSC partners
+  \item Usage is free of charge
+  \item Use your account credentials at your affiliated university to request a VSC-id and connect
+\end{itemize}
+
+
+\subsection{Researchers and staff affiliated with other Flemish or federal research institutes}
+
+\begin{itemize}
+  \item Includes researchers from e.g. INBO, ILVO, RBINS, etc.
+  \item Usage subject to an agreement with Ghent University
```

Comment:
This implies that this whole chapter should only be included conditionally with an `/ifgent`?

Reply:
this chapter is in a /gent folder, so I assumed this was already the case

Reply:
Looks like it is, this chapter is included via a special `\includesite`